code/game/objects/items/binoculars.dm

```diff
@@ -214,9 +217,13 @@
 					QDEL_NULL(laser)
 					break
 		if(MODE_RANGE_FINDER)
+			if(!linked_mortar)
+				return
 			targetturf = TU
-			to_chat(user, "<span class='notice'>COORDINATES: LONGITUDE [targetturf.x]. LATITUDE [targetturf.y].</span>")
+			to_chat(user, "<span class='notice'>COORDINATES TARGETED: LONGITUDE [targetturf.x]. LATITUDE [targetturf.y].</span>")
 			playsound(src, 'sound/effects/binoctarget.ogg', 35)
+			SEND_SIGNAL(linked_mortar, COMSIG_MORTAR_TARGET, TU, user)
```

Comment:
I know i told you to use signals, but since you kept having linked_mortar as a reference, there is no reason to do signals rather than proc here

Reply:
no i know, this code doesnt work and i was using this to try and figure signals out a bit